packages/trpc/server/routers/viewer/webhook/util.ts

```diff
@@ -10,41 +11,132 @@ export const webhookProcedure = authedProcedure
   .use(async ({ ctx, input, next }) => {
```

Comment:
Authorized user:

- If we have an `id` the webhook needs to belong to the user or the user has to be an owner/admin of the team that owns that webhook. 

- If we do not have an `id` we get a `teamId` or an `eventTypeId`. If we have `teamId` the user needs to be the owner/admin of that team. If we have `eventTypeId` the user needs to own the event type or is the owner/admin of the team that owns that event type